src/adagrams.js

```diff
@@ -1,15 +1,193 @@
+
+
+
+const letterPool = {
```

Comment:
We should name the variable with all caps and underscores to indicate that this is a constant variable LETTER_POOL